doc/Development_Documentation/18_Tools_and_Features/25_System_Settings.md

```diff
@@ -123,3 +123,32 @@ Settings for outbound HTTP connectivity of Pimcore - needed e.g. for Pimcore Upd
 Possibility for configuring different newsletter delivery settings from the default e-mail settings.
  
  
+## Access system config in PHP Controller
+Using `\Pimcore\Config::getSystemConfig()` is deprecated. You can choose one of the following options to access the system configuration:
+
+```php 
+<?php
+
+namespace AppBundle\Controller;
+
+use Pimcore\Controller\FrontendController;
+use Symfony\Component\HttpFoundation\Request;
+use Pimcore\Config;
+
+class DefaultController extends FrontendController
+{
+ 
+    /**
+     * @param Request $request
```

Comment:
I'd say the DocBlock should either be removed or `@param Config $config` should be added here.

Reply:
Oh, it's already merged...

Reply:
👍 thanks, will push that straight to master